### ISSUE TITLE
issue.addComment does not allow visibility

### ISSUE THREAD

**Issue by Mustek** [floralvikings/jira-connector#31]
Per the Jira api docs,

addComment can have the following payload:

```
    "body": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque eget venenatis elit. Duis eu justo eget augue iaculis fermentum. Sed semper quam laoreet nisi egestas at posuere augue semper.",
    "visibility": {
        "type": "role",
        "value": "Administrators"
    }
```

PR #25 prevents the visibility from being sent by only sending the body string. Your documentation also expects it to be an object, not a string.

Please revert #25 to its previous state.

**Comment by floralvikings** [comment]
You're absolutely correct.  Rather than revert, though, I think it would be better to allow either invocation; since (I'd imagine) most comments don't require explicitly setting visibility, it definitely *does* make sense to allow a simple string argument rather than requiring it to be wrapped.  Also, it wouldn't break compatibility with existing usage.

Sound acceptable?  If so, I'll try and get it worked out and published this weekend.

**Comment by Mustek** [comment]
That would probably be the best of 2 worlds.

**Comment by floralvikings** [comment]
Published as 2.2.0; let me know if it's not what you were after.

Thanks!
